sys/posix/include/semaphore.h

```diff
@@ -87,7 +64,7 @@ int sem_wait(sem_t *sem);
  * @brief WARNING: currently not supported
  *
  * @param sem Semaphore to wait on
- * @param abstime Max time to wait for a post
+ * @param abstime Absolute time to wait for a post
```

Comment:
I'll try to extract the meaning from the POSIX specification (in general, the doc in this file needs some fixing)